recipes/sdl_ttf/all/conanfile.py

```diff
@@ -51,7 +51,7 @@ def layout(self):
 
     def requirements(self):
         self.requires("freetype/2.12.1")
-        self.requires("sdl/2.26.1")
+        self.requires("sdl/2.26.5")
```

Comment:
```suggestion
        self.requires("sdl/2.26.5", transitive_headers=True)
```

Reply:
I've added this since it looks the sdl-ttf headers would bring the sdl ones transitively, but there may also be an argument for the test package to direectly require sdl as well if the code in the test package makes _direct_ calls to SDL functionality (rather than via sdl-ttf)